.docker/scripts/init-ext.sh

```diff
@@ -27,11 +27,22 @@ install_requirements () {
 
 . ${APP_DIR}/scripts/activate
 
-install_requirements . dev-requirements requirements-dev
+install_requirements . extensions
```

Comment:
i'm uncertain on this split to use extensions file.

Reply:
the way the install_requirements function works is that it looks through the file patten given and if this is a -$PYTHON_VERSION.txt it will use that and break loop.